src/ros_scanner_node.cpp

```diff
@@ -92,11 +94,11 @@ sensor_msgs::LaserScan ROSScannerNode::buildRosMessage(const LaserScan& lasersca
   sensor_msgs::LaserScan ros_message;
   ros_message.header.stamp = ros::Time::now();
   ros_message.header.frame_id = frame_id_;
-  ros_message.angle_min = degToRad(laserscan.min_scan_angle_ / 10.0);
-  ros_message.angle_max = degToRad(laserscan.max_scan_angle_ / 10.0);
+  ros_message.angle_min = degToRad(laserscan.min_scan_angle_ / 10.0) - degToRad(x_axis_rotation_);
```

Comment:
We could move the `x_axis_rotation_` subtraction into the first `degToRad` function thereby saving a function call